src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.CodeModification/Helpers/ProjectModifierHelper.cs

```diff
@@ -545,6 +545,10 @@ internal static void ApplyReplacementsOnFileOnDisk(string filePath, IEnumerable<
                 }
 
             }
+            else if (change.Prepend)
+            {
+                sourceFileString = change.Block + sourceFileString; // Prepending block to start of file
```

Comment:
This makes it easier to insert a string at the beginning of a file from within a "Replacements" block in the JSON changes configruation.